<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QSvn</class>
 <widget class="QMainWindow" name="QSvn" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QSvn</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <widget class="QSplitter" name="splitterHorizontal" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitterVertical" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeView" name="treeViewWorkingCopy" >
        <property name="selectionMode" >
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
       </widget>
       <widget class="QTreeView" name="treeViewFileList" >
        <property name="selectionMode" >
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="" >
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="buttonCancel" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="editStatusText" />
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
